core/src/main/scala/org/apache/spark/metrics/source/AccumulatorSource.scala

```diff
@@ -64,10 +64,11 @@ class DoubleAccumulatorSource extends AccumulatorSource
  */
```

Comment:
Changes in this file seems to be not necessary if removing metrics by exact match from the shared registry as described above.